.github/workflows/cross-compiles.yml

```diff
@@ -0,0 +1,66 @@
+name: Cross Compile
+
+on: [pull_request, push]
+
+jobs:
+  cross-compilation:
+    strategy:
+      fail-fast: false
+      matrix:
+        # The platform matrix strings are two fields separated with a colon.
+        # The first field is the package to install using apt.
+        # The second field is passed to Configure as command line arguments.
```

Comment:
You might want to edit this comment :wink:

Reply:
Done.